linker/shared/src/test/scala/org/scalajs/linker/AnalyzerTest.scala

```diff
@@ -139,14 +138,16 @@ class AnalyzerTest {
     def kindsBaseFor(kindSub: ClassKind): Seq[ClassKind] = {
       import ClassKind._
       kindSub match {
-        case Class | ModuleClass | HijackedClass =>
+        case Class | ModuleClass =>
           Seq(Interface, ModuleClass, JSClass, NativeJSClass)
         case Interface =>
           // interfaces are checked in the ClassDefChecker.
           throw new AssertionError("unreachable")
         case JSClass | JSModuleClass | NativeJSClass | NativeJSModuleClass |
             AbstractJSType =>
           Seq(Class, Interface, AbstractJSType, JSModuleClass)
+        case HijackedClass =>
+          throw new AssertionError("Cannot test HijackedClass before it fails earlier")
```

Comment:
I think the text in the assertion error got garbled, probably during a reformulation?

Reply:
Ah it should have been "because" instead of "before".